src/components/ToastTextToCad.tsx

```diff
@@ -411,3 +415,68 @@ function traverseSceneToStyleObjects({
     }
   })
 }
+
+export function ToastPromptToEditCadSuccess({
+  toastId,
+  token,
+  data,
+  oldCode,
+}: {
+  toastId: string
+  oldCode: string
+  data: TextToCadIteration_type
+  token?: string
+}) {
+  const modelId = data.id
+
+  return (
+    <div className="flex gap-4 min-w-80">
+      <div className="flex flex-col justify-between gap-6">
+        <section>
+          <h2>Prompt to edit successful</h2>
+          <p className="text-sm text-chalkboard-70 dark:text-chalkboard-30">
+            Prompt: "
+            {data?.prompt && data?.prompt?.length > PROMPT_TRUNCATE_LENGTH
+              ? data.prompt.slice(0, PROMPT_TRUNCATE_LENGTH) + '...'
+              : data.prompt}
+            "
+          </p>
+          <p>Do you want to keep the change?</p>
+        </section>
+        <div className="flex justify-between gap-8">
+          <ActionButton
+            Element="button"
+            iconStart={{
+              icon: 'close',
+            }}
+            data-negative-button={'reject'}
+            name={'Reject'}
+            onClick={() => {
+              sendTelemetry(modelId, 'rejected', token).catch(reportRejection)
+              codeManager.updateCodeEditor(oldCode)
+              kclManager.executeCode().catch(reportRejection)
+              toast.dismiss(toastId)
+            }}
+          >
+            {'Reject'}
+          </ActionButton>
+
+          <ActionButton
+            Element="button"
+            iconStart={{
+              icon: 'checkmark',
+            }}
+            name="Accept"
+            onClick={() => {
+              // eslint-disable-next-line @typescript-eslint/no-floating-promises
+              sendTelemetry(modelId, 'accepted', token)
```

Comment:
Judging by the Reject button I think you just need a `.catch` to make this not need an ignore comment